recipes/nsls2-analysis/meta.yaml

```diff
@@ -0,0 +1,217 @@
+{% set name = "nsls2-analysis" %}
+{% set year = "2021" %}
+{% set cycle = "2" %}
+{% set version = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ year }}C{{ cycle }}.{{ version }}
+
+build:
+  number: 0
+  skip: true  # [win and osx]
+
+requirements:
+  host:
+    - python
+  run:
+    - python
+    - amostra <=1.0
+    - ansiwrap
+    - area-detector-handlers
+    - arvpyf
+    - attrs >=18.0
+    - black
+    - bluesky >=1.6.7
+    - bluesky-kafka  # [not win]
+    - bluesky-live
+    - boto3
+    - chxtools
+    - conda-pack
+    - csxtools
+    # TODO: check that the latest 'distributed' package works with the latest
+    # 'dask' package. Currently we are pinning the both libraries to have the
+    # same version.
+    - dask ={{ dask_pin }}
```

Comment:
The `dask_pin` variable is not defined. That should be added on the top of the file next to other Jinja2 vars.